### ISSUE TITLE
Add thumbnail for Apply Functions Cheat Sheet to README

### ISSUE THREAD

**Issue by batpigandme** [tidyverse/purrr#467]

**Comment by hadley** [comment]
Do we have the thumbnails somewhere? Or a script to make them?

**Comment by batpigandme** [comment]
Yeah, there are thumbs for all of them here: https://github.com/rstudio/cheatsheets/tree/master/pngs , but I can pull and resize them as desired.

**Comment by batpigandme** [comment]
purrr's is here: https://raw.githubusercontent.com/rstudio/cheatsheets/master/pngs/purrr.png

**Comment by hadley** [comment]
Yeah, that seems a bit big. Maybe half that size? (so 300px wide, but 600px for retina?)

**Comment by batpigandme** [comment]
<img src = "https://user-images.githubusercontent.com/831732/39709032-bc10f3f4-51e6-11e8-9d3b-fd65a14f138a.png" style="max-width:300px; height:auto;" />

**Comment by batpigandme** [comment]
Sounds good. Will it bork the pgkdown rendering if I put the image file directly in `/docs/`?

**Comment by hadley** [comment]
Yeah, we need to think about where it will live. Maybe in `man/figures/`? It needs to be there if we want to put in the README anyway.

**Comment by batpigandme** [comment]
Where in the README do you want these to go?

**Comment by hadley** [comment]
You tell me!
